gads-ui/src/hooks/useAdmin.js

```diff
@@ -0,0 +1,186 @@
+import { useState } from 'react'
+
+import { api } from '../services/api'
+
+export function useAdmin() {
+    const [providers, setProviders] = useState([])
+
+    const listProviders = async () => {
+        try{
+            const response = await api.get(`/admin/providers`)
+
+            if(response.status === 200) {
+                let providers = response.data ?? {}
+                setProviders(providers)
+                return providers
+            }else {
+                return {
+                    success: false,
+                    message: 'An unknown error has occurred.',
+                    response: response.data
+                }
+            }
+        }catch(error) {
+            if(error.response) {
+                if(error.response.status === 403) {
```

Comment:
I don't believe I am returning 403 anywhere on the backend right now, why are we handling it?